R/check.R

```diff
@@ -39,17 +39,18 @@
 #'
 #' @return An object containing errors, warnings, and notes.
 #' @template devtools
+#' @inheritParams rcmdcheck::rcmdcheck
 #' @param document By default (`NULL`) will document if your installed
 #'   roxygen2 version matches the version declared in the `DESCRIPTION`
 #'   file. Use `TRUE` or `FALSE` to override the default.
 #' @param build_args Additional arguments passed to `R CMD build`
-#' @param check_dir the directory in which the package is checked
-#'   compatibility. `args = "--output=/foo/bar"` can be used to change the
-#'   check directory.
 #' @param ... Additional arguments passed on to [pkgbuild::build()].
 #' @param vignettes If `FALSE`, do not build or check vignettes, equivalent to
 #'   using `args = '--ignore-vignettes' and `build_args = '--no-build-vignettes'.
-#' @param cleanup Deprecated.
+#' @param cleanup `r lifecycle::badge("deprecated")` `check()` no longer does
+#'   any cleanup, so if you want to inspect the built package, store the return
```

Comment:
It seems worth mentioning that you should store the return value of `check()`, so that garbage collection doesn't happen right away.

Reply:
In my experiments (see enumeration point 1 in https://github.com/r-lib/devtools/issues/2448#issuecomment-1206127954), I didn't have to store the return value of `check()` to ensure that the `.Rcheck` directory is not deleted.

Reply:
I think the part about storing the return value can be omitted from the docs, given https://github.com/r-lib/devtools/pull/2455#discussion_r940911269.